fastplotlib/graphics/line.py

```diff
@@ -83,3 +83,35 @@ def __init__(
 
         if z_position is not None:
             self.world_object.position.z = z_position
+
+    def _set_feature(self, feature: str, new_data: Any, indices: Any = None):
+        if not hasattr(self, "_previous_data"):
+            self._previous_data = {}
```

Comment:
better to create a dict or any iterable types explicitly like `dict()`, the `{}` can be buggy in a few very strange situations

`{}` is technically faster than `dict()`, but the difference is in nanoseconds